### DISCUSSION TITLE
Integration with Nova? or Workarounds?

### DISCUSSION THREAD

**Discussion by DevHoracioRodriguez** [livewire/livewire#5598] [1. Help]
Laravel 10.x
Nova 4.0
Livewire 2.x

1. Are there plans to integrate with Nova?
2. If not, is there any workarounds? As an example I'm trying to make a custom field...

**Comment by joshhanley** [comment]
@CyberEkklesiaOwner no plans. I'd suggest using Nova and Vue to create your custom field rather than trying to get Livewire to work inside Nova (which is built on VueJS).

The other option is to use Livewire based admin panel, such as Filament Admin https://filamentphp.com/

Hope this helps!
